docs/guide/configuration/devices-groups.md

```diff
@@ -98,6 +98,15 @@ and `debounce` option without `debounce_ignore` publishes only last payload with
 hand `debounce: 1` with `debounce_ignore: - action` will publish all unique action messages, at least two (
 e.g. `action: rotate_left` and `action: rotate_stop`)
 
+**`throttle`**  
+Throttle messages of this device. When setting e.g. `throttle: 10` the first message from device is accepted but all other messages in next 10 seconds will be ignored. Be carefull when using this option, unlike debounce that can ignore some attributes this will drop entire message.
+
+For now this option will only work when debounce is disabled.
+
+This is handy for some sensor devices that keep sending messages each second, e.g. 'TS0601_air_quality_sensor', even without considerable changes of values, like some water level sensors and ambient sensors.
+
+This option will reduce the MQTT messages sent for this device and also reduce storage if the states are stored in homeassistant.
```

Comment:
```markdown
Setting this option reduces the number of MQTT messages sent for a particular device. This is directly linked to how high the option is set. When used on misbehaving devices with proper values, it can drastically reduce the size of external databases that store history (like Home Assistant), and yet have little to no impact on the quality of said history.
```

Reply:
@Nerivec here you think that I should replace both paragraphs with your suggestion or only the second one ?

Reply:
I'd say both, you can put some example in parenthesis
```markdown
misbehaving devices (some water level, presence, etc. sensors, e.g. `TS0601_air_quality_sensor`)
```

Reply:
@Nerivec for me reading the example in separate became easier to understand. I'm not an english native. 
can you check my last commit ?